CMakeLists.txt

```diff
@@ -194,6 +194,21 @@ elseif(CMAKE_SYSTEM_PROCESSOR MATCHES "^(riscv)")
     check_cxx_compiler_flag("-march=rv64gcv_zfh" NCNN_COMPILER_SUPPORT_RVV_FP16)
 
     if(NCNN_COMPILER_SUPPORT_RVV)
+        include(${CMAKE_CURRENT_SOURCE_DIR}/cmake/ncnn_check_rvv_vfredsum.cmake)
+        if(NOT NCNN_COMPILER_USE_VFREDSUM AND NCNN_COMPILER_SUPPORT_RVV_INTRINSIC)
+            if(NCNN_COMPILER_USE_VFREDUSUM)
+                SET(NCNN_REPLACED_VFREDSUM vfredusum)
+            else()
+                SET(NCNN_REPLACED_VFREDSUM vfredosum)
```

Comment:
It seems that some assemblers are unable to convert `vfredusum` into corresponding binary.